### ISSUE TITLE
Implement the Written Emergency Information Tutorial

### ISSUE THREAD

**Issue by AricaLConrad** [MobileApps-Cascadia/TechEase_iOS#10]
Description:
Now that I finished writing and rewriting the Emergency Information tutorial, it is time to implement the written tutorial in the app. The Emergency Information tutorial is split into four parts and will need four buttons to go to each tutorial part. Each tutorial part will have the text and screenshots that I wrote and created for it.

Definition of Done:
When there are four buttons that go to each part of the Emergency Information tutorial and all the text and screenshots are present.

**Comment by AricaLConrad** [comment]
5/4/2022 Progress:

I started implementing the Emergency Information tutorial today. I was able to create four buttons for each tutorial part, and each button navigates to its respective tutorial part screen. I was able to use Mackenzie's file I/O code to add the text for Part 1 of the tutorial.

I am currently having trouble with the four tutorial part buttons, though. The code specifies that the design of the button fills the available space. This means the four buttons are an inconsistent size, since they are dependent on the space made available after their respective tutorial part's informational text. I am hesitant to use an absolute button size, due to wanting relative button sizes across all screen sizes, but I may end up needing to go this route.

I will continue to research and experiment with the button design. I may want to use a different screen layout altogether.

**Comment by AricaLConrad** [comment]
This issue is now complete! I finished implementing the Emergency Information written tutorial and also saved all the .txt and image files in preparation for the Calendar and Compass written tutorials.

I had some difficulty saving the image and .txt files initially. Xcode gave me warnings when I added the images, because the file extension was "PNG." When I changed the file extension to "png," the warnings went away. I struggled with the .txt files as well because I was adding the files to the project incorrectly. Once I specified "Copy items if needed," I was able to see the .txt files properly in the file directory.

I used the following resources to solve my problems:

- https://stackoverflow.com/questions/32632577/xcode-7-warning-the-launch-image-does-not-have-a-valid-extension?msclkid=97c53568cf3811ec8ebc9d66111e73f3 (Answer 3)
- https://www.dev2qa.com/how-to-add-resource-files-into-xcode-project-and-ios-app/?msclkid=3541f0f8cf2011ec9e0db919f4952643 (Option 1)
